src/Boilerplates.php

```diff
@@ -0,0 +1,99 @@
+<?php
+
+namespace JustCoded\WP\Composer;
+
+use Composer\IO\IOInterface;
+use Composer\Script\Event;
+
+class Boilerplates {
+
+	/**
+	 *  Arguments array
+	 * @var $args
+	 */
+	public static $args = array();
+
+
+	/**
+	 * @param Event $event
+	 *
+	 * @return bool
+	 */
+	public static function theme( Event $event ) {
+		$io = $event->getIO();
+
+		$theme_name = str_replace( 't=', '', $event->getArguments()[1] );
+		$theme_slug = str_replace( ' ', '', $theme_name );
+		$theme_dir  = str_replace( ' ', '', $event->getArguments()[0] );
+		if ( empty( $theme_dir ) ) {
+			$theme_dir = 'wp-content/themes';
+		}
```

Comment:
wrong parameters getter.

folder/slug - is a trim(1st parameter without keys)
title  - you need to find argument with -t
namespace - you need to find argument with -ns
destination dir    - you need to find argument with -f

it's better to make helper, which can parse event arguments and provide array like:

['key' => value, ... (for options), 0 => value, 1 => value (for params without options)]

Reply:
ready